hapic/util.py

```diff
@@ -1,53 +1,91 @@
 # -*- coding: utf-8 -*-
 import typing
 
+from hapic.exception import HapicException
+
 LOGGER_NAME = "hapic"
 
 
-class LowercaseDictKeys(dict):
+class LowercaseKeysDict(dict):
     """
     Like a dict but try to use lowercase version of given keys.
-    Must give lowercase key to ths dict when fill it.
+    Must give string lowercase key to ths dict when fill it.
     """
 
-    def get(
-        self, key: typing.Any, default_value: typing.Any = None
-    ) -> typing.Any:
+    @staticmethod
+    def check_key(key: str) -> None:
+        try:
+            if key.lower() != key:
+                raise HapicException(
```

Comment:
Should raise an exception inheriting from `HapicException`, like `NotLowercaseCaseException` ?

Reply:
You're right. It is done.